src/client/components/LeftBlueBar/LeftBlueBar.styles.css

```diff
@@ -43,12 +43,15 @@
   border-left: 16px solid rgb(230, 97, 20);
   border-bottom: 8px solid transparent;
   margin-left: 15px;
+  margin-top: 6px;
 }
 
 .login {
   display: flex;
+  justify-content: center;
   align-items: center;
   padding-top: 57px;
+  margin: 0 auto;
```

Comment:
`justify-content: center` and `margin: 0 auto` do the same, so you can remove fx. `margin `